Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -51,23 +53,46 @@
- BTWK_story_event.0115.desc_norwenna_dead:0 "With Norwenna not long passing after her husband - the late Prince Mordred - Uther's intention of an alliance forged through marriage was not to be, leaving Dumnonia more exposed than he would've liked. However, some loyalists to the High King and his young heir were still to be found for its defence. Chiefly among them is Tewdric, King of Gwent, the legendary druid Merlin, and Uther's own son, the renowned warlord Prince Arthur. While Uther was unsuccessful in uniting the British warlords towards their continued survival, this little web of alliances will at least give Dumnonia some reprieve from the assaults of its enemies - for a time at least."
+ BTWK_story_event.0115.desc_header:0 "The High King of Britannia and King of [GetTitleByKey('k_dumnonia').GetNameNoTier], Uther Pendragon, has recently hosted a High Council in the old Roman town of Glevum."
+ BTWK_story_event.0115.desc_alliance:0 "Word has reached you that through arranging a marriage between Norwenna - the widow of the late Prince of Dumnonia, Mordred - and [lady_norwenna_husband.GetShortUINameNoFormat], Uther has secured an alliance between his realm, and [dumnonia_alliance_realm.GetNameNoTier]. A number of guardians have also been sworn in by oath to protect the new Edling Mordred. Chiefly among them is [GetTitleByKey('k_gwent').GetHolder.GetShortUINameNoFormat], King of [GetTitleByKey('k_gwent').GetNameNoTier], the legendary druid [GetTitleByKey('c_avalon').GetHolder.GetShortUINameNoFormat], and Uther's bastard son - the renowned warlord Arthur. Such a web of alliances will surely fortify Dumnonia against its enemies until the young Prince reaches maturity."
+ BTWK_story_event.0115.desc_arthur:0 "Word has reached you that, despite the misgivings he holds for his bastard son, Uther has arranged a marriage between Arthur, and his late brother's widow, Norwenna. With Arthur's arrangment as the Edling Prince's foster father, his return to the shores of Britannia from the Armorican frontlines is imminent. A number of guardians have also been sworn in by oath to protect the new Edling Mordred. Chiefly among them are [GetTitleByKey('k_gwent').GetHolder.GetShortUINameNoFormat], King of [GetTitleByKey('k_gwent').GetNameNoTier], the legendary druid [GetTitleByKey('c_avalon').GetHolder.GetShortUINameNoFormat]. While Uther was unsuccessful in uniting the British warlords towards their continued survival, the impending return of Arthur may be enough to give Dumnonia some pause from the assaults of its enemies."
```

Comment:
1. "Uther has arranged a marriage between Arthur, and his late brother's widow" -> "Uther has agreed to a marriage between Arthur and his late brother's widow"
2. "With Arthur's arrangment as the Edling Prince's foster father, his return" -> "As the Edling's new foster father, Arthur's return"
3. "have also been sworn in by oath to protect the new Edling Mordred" -> "have also been sworn to protect the new Edling Mordred with their lives"
4. "While Uther was unsuccessful in uniting the British warlords towards their continued survival" -> "While Uther was unsuccessful in ending the conflicts that have wreaked the island since the end of Roman rule"

Reply:
1. Amended. Could the part of the sentence "and his late brother's widow" construe that Arthur is marrying "[Uther's] late brother's widow"? I know it is mentioned previously that Mordred is the late brother of Arthur, but I thought the sentence read strangely when I went over it just.
2. Amended.
3. Amended.
4. Amended. Changed "wreaked" to "harried", and "the island" to "Britannia".

Reply:
1. Hmm true, it might be read that way. Maybe "...between Arthur and Norwenna, widow of the late heir Mordred"?

Reply:
Agreed. Amended.